graphene_django_plus/mutations.py

```diff
@@ -122,17 +122,20 @@ def _get_fields(model, only_fields, exclude_fields, required_fields, registry):
 
         if required_fields is not None:
             required = name in required_fields
-            f.kwargs["required"] = required
+            if hasattr(f, 'kwargs'):
```

Comment:
consistency with double quotes would be nice (since I see double quotes are used here)

Reply:
Done